Makefile

```diff
@@ -0,0 +1,24 @@
+release:
+	rm -rf dist/
+	python setup.py sdist bdist_wheel
+	twine upload dist/*
+
+isort:
+	isort --remove-import . --apply --use-parentheses --trailing-comma
+
+format:
+	black --line-length 79 .
```

Comment:
Can we use the default black settings? It's 88 characters.

Reply:
Done!